packages/block-editor/src/components/use-setting/index.js

```diff
@@ -50,6 +50,7 @@ const deprecatedFlags = {
 };
 
 const PATHS_WITH_MERGE = {
+	'color.duotone': true,
```

Comment:
When I was searching through the code, I found `PATHS_WITH_MERGE` in [one other place](https://github.com/ajlende/gutenberg/blob/5ec967cc2e3f05f713e2461b6fe9d9eb2e85a4a5/packages/edit-site/src/components/editor/utils.js#L94). Not sure why it's in both places, but we can probably add it there too to be safe.

Reply:
We should consolidate both in https://github.com/ajlende/gutenberg/blob/5ec967cc2e3f05f713e2461b6fe9d9eb2e85a4a5/packages/blocks/src/api/constants.js either in a follow-up or previously to this PR.